modules/KalturaSupport/components/playlistAPI.js

```diff
@@ -397,7 +397,7 @@
 				title = kWidget.sanitize( title );
 				description = kWidget.sanitize( description );
 
-				var thumbnailUrl = item.thumbnailUrl || customData.thumbUrl || this.getThumbUrl(item);
+				var thumbnailUrl = this.getThumbUrl(item) || item.thumbnailUrl || customData.thumbUrl;
```

Comment:
there is a reason for this order, you want it to take it from item or custom level before falling back to default.

Reply:
If issue is KS we need to enforce the `loadThumbnailWithKs` settings option here as well.

Reply:
or maybe handle the item.thumbnailUrl and customData.thumbUrl in `this.getThumbUrl` call